packages/opentelemetry-node/src/instrumentation/PluginLoader.ts

```diff
@@ -83,6 +83,18 @@ export class PluginLoader {
         return this;
       }
 
+      const alreadyRequiredModules = Object.keys(require.cache);
+      const requiredModulesToHook = modulesToHook.filter(name =>
+        alreadyRequiredModules.some(cached => cached.includes(name))
```

Comment:
you could use `find` instead of `some` as `find` will stop after first element is found.